src/Swashbuckle.AspNetCore.Filters/Examples/MvcOutputFormatter.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Net.Http.Headers;
+using System.Linq;
+
+namespace Swashbuckle.AspNetCore.Filters
+{
+    internal class MvcOutputFormatter
+    {
+        private readonly object initializeLock = new object();
+        private bool initializedOutputFormatterSelector;
+
+        private readonly IOptions<MvcOptions> selectorOptions;

```

Comment:
Could this be an MvcOptions instead of an IOptions<MvcOptions>?

Reply:
I think so yes, @ridingwolf we can have `GetSelectorOptions` not wrap it in an IOptions again and just store the MvcOptions